### PR TITLE
Sharpen and msaa effect in default pipeline

### PR DESCRIPTION
**Pull request by TrevorDev**

### CODE REVIEW

File: dist/preview release/what's new.md [hunk 1]
```diff
@@ -80,7 +80,8 @@
 - Support depth maps for multiple active cameras for post processes like depth of field ([trevordev](https://github.com/trevordev))
 - Integrates depth texture support in the engine ([sebavan](https://github.com/sebavan))
 - NPM package now has a dependency system, updated during build. ([RaananW](https://github.com/RaananW))
-- Default pipeline will use webGL 2.0 anti aliasing by default if supported, default fragment shader will clamp negative values to avoid underflow, webVR post processing will render to eye texture size ([trevordev](https://github.com/trevordev))
+- Default fragment shader will clamp negative values to avoid underflow, webVR post processing will render to eye texture size ([trevordev](https://github.com/trevordev))
+- Add msaa and sharpening options to the default pipeline ([trevordev](https://github.com/trevordev))
```
Comment (deltakosh) [inline dist/preview release/what's new.md]: Need a link to a demo:)
And this is major update

Comment (TrevorDev) [reply dist/preview release/what's new.md]: Talking with Seb, we think there are a couple more improvements to the default pipeline. Planned to add chromatic abrasion, grain, improved bloom and fixed ordering of effects. Once that's done I'll update the default pipeline demo.

**Comment by deltakosh** [comment]
Then we are good:)

**Comment by deltakosh** [comment]
Also please note to update doc as well at the end

**Pull request merged**